### DISCUSSION TITLE
How to select icon?

### DISCUSSION THREAD

**Discussion by MasoudRahmani** [DineshSolanki/FoliCon#115] [Q&A] [answered]
Can't select custom icon from professional mode or normal mode.

When i click an icon, image viewer opens and when i click on window button, this is the error i receive.

![image](https://github.com/DineshSolanki/FoliCon/assets/5321268/1f22f532-ae07-43c5-976a-ffaf1a355f54)

![image](https://github.com/DineshSolanki/FoliCon/assets/5321268/acf76100-837b-41c0-bfa9-d9a45cf52081)

**Comment by DineshSolanki** [comment, marked as answer]
you have to double click to select the icon. don't use the window icon.
single click will open the image viewer

**Reply by MasoudRahmani** [reply]
Thanks, It didn't pass my mind. ❤️

**Reply by DineshSolanki** [reply]
No problem, it was an intuitive way for me, I should create a detailed ReadMe on FoliCon's usage, thanks for pointing out.
